build-system/tasks/dep-check.js

```diff
@@ -204,9 +203,12 @@ function getGraph(entryModule) {
  */
 function getEntryModule(extensionFolder) {
   var extension = path.basename(extensionFolder);
-  var versions = extensionsVersions[extension] || ['0.1'];
-  return versions.map(v =>
-      `${extensionFolder}/${v}/${extension}.js`);
+  return fs.readdirAsync(extensionFolder).then(dirItems => {
```

Comment:
I'm not seeing that issue after running `gulp clean` and `gulp dep-check`. Though it does say in the log that it's `Recompiling CSS in amp-sticky-ad`. There were some issues with the paths in my code which I just fixed, might want to test it locally again?